### PR TITLE
fix: uploadList thumbUrl image type check

### LINKED ISSUE #12948: UploadList don't check the file type to create thumbUrl.
### Description
`UploadList` don't check the fileType if I upload a video file, this is the screenshot.

image

detail in [Zhihu](https://zhuanlan.zhihu.com/p/48140299)

### Example
[Codesandbox](https://codesandbox.io/s/mjx860jjyj)

### PR DESCRIPTION
**Pull request by ycjcl868**
Close #12948 .

### CODE REVIEW

**Comment by afc163** [comment]
ci broken

**Comment by ycjcl868** [comment]
broken a lot

File: components/upload/UploadList.tsx [hunk 1]
```diff
@@ -6,8 +6,12 @@ import Progress from '../progress';
 import classNames from 'classnames';
 import { UploadListProps, UploadFile, UploadListType } from './interface';
 
+const imageTypes: string[] = ['image', 'webp', 'png', 'svg', 'gif', 'jpg', 'jpeg', 'bmp'];
 // https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL
 const previewFile = (file: File, callback: Function) => {
+  if (file.type && !imageTypes.includes(file.type)) {
+    callback('');
+  }
```
Comment (zombieJ) [inline components/upload/UploadList.tsx]: Can we add one more cache here to avoid preview file repeat?

Comment (ycjcl868) [reply components/upload/UploadList.tsx]: good idea ~

Comment (ycjcl868) [reply components/upload/UploadList.tsx]: https://github.com/ant-design/ant-design/blob/efe08a8b04d5221fb9c7e059ba7e7ae547c73ee6/components/upload/UploadList.tsx#L82，has avoided preview file repeat if `file.thumbUrl` existed.

**Pull request merged**